content/server/soju.md

```diff
@@ -0,0 +1,194 @@
+---
+## Guide Information
+title: "Soju"
+description: A user-friendly IRC bouncer
+icon:
+date: 2024-06-26
+ports: [6697]
+
+## Author Information
+author: Ángel Castañeda
+email: angel@acsq.me
+matrix: "@acsquared:cyberia.club"
+
+---
+
+Soju is an irc bouncer that turns the protocol into something modern everyone can use with chat history, persistent connections, and multiserver support.
+
+This guide assumes you have a subdomain at `irc.example.org` and are port forwarding at 6697.
+
+## Installation
+
+Soju is not in debian 12's repos, so we'll need to build it from source.
+
+```sh
+VERSION=v0.8.0
+curl -fLO https://git.sr.ht/~emersion/soju/archive/$VERSION.tar.gz
+tar xvf $VERSION.tar.gz
+rm $VERSION.tar.gz
+mv soju-$VERSION soju
+```
+
+Soju's latest release depends on go 1.19 which happens to be the exact same version in debian's repos. Let's install some more depencencies to build our binaries:
+
+```sh
+apt install golang scdoc libsqlite3-dev
+export GOFLAGS="-tags=libsqlite3"
+make
+make install
+```
+
+## Deployment
+
+### Unix User
+
+From the docs: "soju is designed to be run as a system-wide service under a separate user account."
+
+So let's make that separate user.
+
+```sh
+useradd -m -d /var/lib/soju -s /usr/bin/bash soju
```

Comment:
Soju's user doesn't need to have a shell set by default
```suggestion
Let's make that user:

```sh
useradd -m -d /var/lib/soju soju
```